namespaces/_default_.yml

```diff
@@ -60,44 +61,50 @@ _default_:
       type: string
       example: info
       description: >
-        Loging level as provided by: rsyslog(severitytext property), python's logging module, etc.
-        Possible values: info, warning, notice, err, debug
+        Logging level as provided by: rsyslog(severitytext property), python's
+        logging module, etc.
+        Possible values: trace, crit, alert, emerg
```

Comment:
Why not support all of the syslog(3) levels: emerg, alert, crit, err, warn, notice, info, debug?